src/components/media-loader.js

```diff
@@ -65,9 +73,24 @@ AFRAME.registerComponent("media-loader", {
           },
           { once: true }
         );
-        this.el.setAttribute("image-plus", { src: raw, contentType });
+        let blobUrl;
+        if (token) {
+          const imageResponse = await fetch(raw, {
```

Comment:
Hmm why are we doing this idependantly for images/gltfs? Seems like it should just be part of determining what url to load. Those with tokens end up loading a blob url.

Reply:
👍 DRYed it up